universal-login-react/src/ui/styles/emoji.sass

```diff
@@ -313,14 +313,15 @@
     flex-direction: column
 
   .emoji-success-text
```

Comment:
This style will affect text color in jarvis. Universal Login styles should be in default.

Reply:
Fixed